external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -2074,7 +2074,17 @@ fn file_sources(
 pub fn expansion_mod_ident_to_map_key(mod_ident: &E::ModuleIdent_) -> String {
     use E::Address as A;
     match mod_ident.address {
-        A::Numerical { value, .. } => format!("{value}::{}", mod_ident.module).to_string(),
+        A::Numerical {
+            name,
+            value,
+            name_conflict: _,
+        } => {
+            if let Some(n) = name {
+                format!("({n}={value})::{}", mod_ident.module).to_string()
+            } else {
+                format!("{value}::{}", mod_ident.module).to_string()
```

Comment:
Is this shown to the user ever? Or is it just used internally?

Reply:
It is never shown to the user. Only used as a key to a map, accessed both when visiting parsed AST and typed AST (where `ModuleIdentifier`s are different).